components/Logo/Logo.tsx

```diff
@@ -0,0 +1,17 @@
+import styles from "./Logo.module.css";
+
+type LogoProps = {
+  size: string;
+};
+
+function Logo({ size }: LogoProps) {
+  return (
+    <object
+      data="/logorefcast.svg"
+      type="image/svg+xml"
+      className={`${styles[size]}`}
+    ></object>
```

Comment:
```suggestion
    <img
      src="/logorefcast.svg"
      className={`${styles[size]}`}
    />
```
You could solve this with a `img` tag. The `object` tag is rather uncommon